### DISCUSSION TITLE
Windows上的客户端打开窗口不显示出来

### DISCUSSION THREAD

**Discussion by Saul-Github** [nextai-translator/nextai-translator#481] [Q&A] [answered]
昨天还可以用，但今天打开之后，没有窗口出现。任务栏上显示translator在运行，但就是点不开。尝试了不同的版本，都是这种情况

**Comment by wuya1019** [comment]
我也遇到了这个问题，真是急死人了

**Comment by wuya1019** [comment, marked as answer]
找到了 按下面这个方法 删除这2个文件后 重装即可
See https://github.com/yetone/openai-translator/issues/436 见 https://github.com/yetone/openai-translator/issues/436
You can try to delete below folders and reinstall the app (your settings will be lost as a result)
您可以尝试删除以下文件夹并重新安装应用程序（您的设置将因此丢失）

%USERPROFILE%\AppData\Local\xyz.yetone.apps.openai-translator
%USERPROFILE%\AppData\Roaming\xyz.yetone.apps.openai-translator

**Reply by Saul-Github** [reply]
感谢！

**Reply by ShadowsSide** [reply]
> 找到了 按下面这个方法 删除这2个文件后 重装即可 See #436 见 #436 You can try to delete below folders and reinstall the app (your settings will be lost as a result) 您可以尝试删除以下文件夹并重新安装应用程序（您的设置将因此丢失）
> 
> %USERPROFILE%\AppData\Local\xyz.yetone.apps.openai-translator %USERPROFILE%\AppData\Roaming\xyz.yetone.apps.openai-translator

感谢！！！我也遇到了同样的问题，删除文件夹再重装就好了！

**Comment by wuya1019** [comment]
重装好了之后 过段时间又这样了 你们遇到这种情况吗

**Reply by Saul-Github** [reply]
还有这种情况

**Comment by hawkingwan** [comment]
遇到同样的问题，谢谢~~·

**Comment by hawkingwan** [comment]
有人遇到这个情况吗，就是版本升级之后，桌面端软件，窗口不显示出来啦。

**Comment by hawkingwan** [comment]
今天又遇到同样的问题 ~~

**Comment by yetone** [comment]
十分抱歉，最新版已经解决这个问题
